avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1398,7 +1408,7 @@ def polygon2Raster(demHeader, Line, relTh=''):
     # for this we need to know if the path is clockwise or counter clockwise
     # to decide if the radius should be positif or negatif in contains_points
     is_ccw = geoTrans.isCounterClockWise(path)
-    r = 0.001
+    r = 0.000
```

Comment:
This is a setting and needs to go to the `ini` file

Reply:
made an issue